hack/update_crdb_versions.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+)
+
+const INVERTED_VERSIONS_REGEXP = "^v19|^v21.1.8$|latest|ubi$"
```

Comment:
Because this lives directly under hack, these constants might end up conflicting with other binaries over time (ok, probably not this one, but `URL` for example).

Here are a few ideas that might help:

1. Prefix them with `crdbVersions`
2. Define them inside the methods the use them (still as `const`)
3. Move this into it's own package (i.e. hack/update_crdb_versions/main.go)

Reply:
went with crdbVersions prefixes